File: app/src/main/java/com/example/whattodo/model/Ofert.java
package com.example.whattodo.model;

public class Ofert {

    private String photo, title, event, localitzacio, validesa;
    private boolean adquirit;

    public Ofert(){}

    public Ofert(String photo, String title, String event, String localitzacio, String validesa, boolean adquirit){
        this.photo = photo;
        this.title = title;
        this.event = event;
        this.localitzacio = localitzacio;
        this.validesa = validesa;
        this.adquirit = adquirit;
    }

    public String getPhoto(){
        return photo;
    }

    public String getTitle() {
        return title;
    }

    public String getEvent() {
        return event;
    }

    public String getLocalitzacio() {
        return localitzacio;
    }

    public String getValidesa() {
        return validesa;
    }

    public boolean getAdquirit() {
        return adquirit;
    }

    public void setAdquirit(boolean adquirit){
        this.adquirit = adquirit;
    }
}

File: app/src/main/java/com/example/whattodo/model/Ofert.kt
package com.example.whattodo.model

class Ofert {
    var photo: String? = null
        private set
    var title: String? = null
        private set
    var event: String? = null
        private set
    var localitzacio: String? = null
        private set
    var validesa: String? = null
        private set
    var adquirit = false
    var price: String? = null
        private set

    constructor() {}
    constructor(photo: String?, title: String?, event: String?, localitzacio: String?, validesa: String?, adquirit: Boolean, price: String?) {
        this.photo = photo
        this.title = title
        this.event = event
        this.localitzacio = localitzacio
        this.validesa = validesa
        this.adquirit = adquirit
        this.price = price
    }

}